docs/scenarios/solution-authoring/target-dotnetcore-with-msbuild.md

```diff
@@ -0,0 +1,120 @@
+Using MSBuild to build .NET Core projects
+=========================================
+
+by [Daniel Plaisted](https://github.com/dsplaisted)
+
+The .NET Core tooling is going to [move from project.json to MSBuild based projects](https://blogs.msdn.microsoft.com/dotnet/2016/05/23/changes-to-project-json/).
+We expect the first version of the .NET Core tools that use MSBuild to ship along with Visual Studio "15".  However, it is possible to use MSBuild for .NET Core
```

Comment:
Instead of Visual Studio "15", I'd say next version of Visual Studio. Not all our customers know our internal codenames and could be confusing since the current one is 2015.